HLTriggerOffline/Higgs/python/hltHiggsValidator_cfi.py

```diff
@@ -265,7 +265,7 @@
             "HLT_L1_TripleJet_VBF_v"
             ),
         recJetLabel  = cms.string("ak4PFJetsCHS"),
-        jetTagLabel  = cms.string("pfCombinedSecondaryVertexBJetTags"),
```

Comment:
@jasperlauwers, but what is the actual intention here? I'm a bit surprised that `pfCombinedSecondaryVertexBJetTags` was recommended at any point for Run 2 since this is an outdated tagger based on a pre-Run 1 training. In addition, it was also know, or should have been know, that the default choice for the secondary vertices would be the Inclusive Vertex Finder. Hence, is the intention here to really keep monitoring `pfCombinedSecondaryVertexV2BJetTags`, which uses the new CSVv2 training and AVR (not IVF) SVs, or you actually want to monitor `pfCombinedInclusiveSecondaryVertexV2BJetTags`, which is currently supported by the BTV POG, in all of the cases. Waiting for reply before making any changes. Thanks.